app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="row">
+<%= link_to "Create", new_user_recipe_path(user_id: current_user.id), class: "btn btn-outline-success mb-4 col-sm-3 offset-sm-9" %>
+</div>
+<% @recipes.each do |recipe| %>
+    <div class="card mb-3">
+      <div class="row p-3">
+        <a href="<%= user_recipe_path(:user_id => current_user.id, :id => recipe.id) %>">
+          <div class="col-sm-3">
+            <h2> Recipe <%= recipe.id %> </h2>
+            <%= link_to "Delete", user_recipe_path(user_id: current_user.id, id: recipe.id), class: "btn btn-outline-danger", "data-turbo-method": :delete %>
```

Comment:
### Recipe Index Page
![image](https://user-images.githubusercontent.com/93039359/219692090-80b9ccae-63a5-48a9-ba9d-db3099b072d9.png)
### Recipe Show Page
![image](https://user-images.githubusercontent.com/93039359/219692264-4e476f6c-3067-4dad-9e4e-15430da4b6aa.png)


- [ ] When I made a Recipe with the name of **Chicken Karahi** then it showed my **Recipe 1** on the recipe index page which is not the project requirement :fearful: It would be nice if it shows me **Chicken Karahi** instead of **Recipe 1** I know you followed the wireframe but that's not what wireframe is wanting you to follow :+1: Wireframe is there to just help you with the overall template but not with the actual data in it :+1:

Reply:
Updated

Reply:
resolved! ✔️